src/main/kotlin/ecommerce/controller/ProductController.kt

```diff
@@ -0,0 +1,58 @@
+package ecommerce.controller
+
+import ecommerce.model.Product
+import ecommerce.service.ProductService
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.DeleteMapping
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.PutMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RestController
+import java.net.URI
+
+@RestController
```

Comment:
I see you've used both `@RestController` and `@Controller` in your controllers!
What are the differences between these two annotations?